packages/web3-providers/src/twitter/index.ts

```diff
@@ -31,6 +31,13 @@ function getCSRFToken() {
     return value
 }
 
+async function uploadToTwitter(url: string, config: RequestInit) {
```

Comment:
Move this to `packages/web3-providers/src/helpers.ts`. And rename it as `contentFetch`.